third_party/BUILD

```diff
@@ -44,16 +44,12 @@ nginx_suite(
     tests = glob(
         ["nginx-tests/*.t"],
         exclude = [
-            # limit_req.t is excluded from the test suite because we use 429
-            # instead of 503 as response status code
-            "nginx-tests/limit_req.t",
-            # ssl2.t is excluded from the test suite because boring ssl
-            # does not support SSL_get0_raw_cipherlist used by $ssl_ciphers
-            # variable required by the test.
-            "nginx-tests/ssl2.t",
-            # ssl_sni_reneg.t is excluded from the test suite because boring ssl
-            # does not support renegotiation feature required by the test.
-            "nginx-tests/ssl_sni_reneg.t",
+            # ssl.t, ssl_certificates.t, ssl_stapling are excluded from the
+            # test suite because boring ssl does not support renegotiation
```

Comment:
`s/boring ssl/BoringSSL/`

Also, those tests are failing for reasons other than renegotiation, i.e. missing ciphers variable, missing support for multiple certificate types in the same context and missing support for OCSP stapling.